### PR TITLE
Added .getBytes() method in Java Strings terms folder

### LINKED ISSUE #1582: [Term Entry] Java Strings: `.getBytes()` Method
### Reason/inspiration (optional)

Add a new term entry under the [Java: Strings](https://www.codecademy.com/resources/docs/java/strings) concept. This entry should include:

- An explanation of the method.

- A "Syntax" section with a ```pseudo code snippet showing the general syntax.

- An "Example" section with a ```java code snippet showing how it works.

A markdown file named "getBytes.md" should be added to a folder named "getBytes" and placed in the terms folder located under content/java/concepts/strings/terms

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by AshutoshVJTI**
### Description

- Added new entry for Java Strings: .getBytes()
- Closes #1582 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by AshutoshVJTI** [comment]
@SSwiniarski I have updated the file with suggested change. Thanks!

**Review by SSwiniarski — APPROVED** [review]
@AshutoshVJTI Looks good for a second review.

File: content/java/concepts/strings/terms/getBytes/getBytes.md
Comment (yangc95) [inline content/java/concepts/strings/terms/getBytes/getBytes.md:14]:
```suggestion
The **`.getBytes()`** method encodes the string into an array of bytes that represent the characters in the string.
```

File: content/java/concepts/strings/terms/getBytes/getBytes.md
Comment (yangc95) [inline content/java/concepts/strings/terms/getBytes/getBytes.md:19]:
```suggestion
str.getBytes();
```

File: content/java/concepts/strings/terms/getBytes/getBytes.md
Comment (yangc95) [inline content/java/concepts/strings/terms/getBytes/getBytes.md:37]: Can we use a different name for `str` and `byteArray` here? Perhaps `myStr` and `myByteArray` could work. Feel free to adjust.

**Review by yangc95 — COMMENTED** [review]
@AshutoshVJTI Great and clear entry! I did leave two minor comments after which I think this entry is good for merge 🙂

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks @AshutoshVJTI for your positivity throughout this process. Great to have you on Docs! Here's the live link: https://www.codecademy.com/resources/docs/java/strings/getBytes

Looks great:
Screenshot 2023-03-27 at 6 07 22 PM

cc: @KTom101 @SSwiniarski

**Pull request merged**
